src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_PRICE = 40;
+  const LONG_TERM = 7;
+  const MIN_TERM = 3;
+  const MIN_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const renCost = DAY_PRICE * days;
+
+  if (days >= MIN_TERM && days < LONG_TERM) {
```

Comment:
You can put this condition after the second one. And change it to just days >= MIN_TERM

Reply:
fixed))))